recipes/nmslib/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "nmslib" %}
+{% set version = "2.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/8b/ae/168a8cecf02531b06140d66bdd6e3f05c3fa668ccb5f77369c37e31009eb/nmslib-2.0.5.tar.gz
+  sha256: 6a72b132b4b7fe7d697767005c69bc66c8fbc83dbba2d07109c0da29c0f00353
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
Should it be possible to build this on windows?

Reply:
Yes... it builds for windows too now.